Makefile

```
@@ -70,5 +71,12 @@ run-mypy: ## Run MyPy in Container
 run-local-lineage-server: ## Run flask based local Lineage server
 	FLASK_APP=dev/local_flask_lineage_server.py flask run --host 0.0.0.0 --port 5050
 
+test-rc-dependencies: ## Tests providers RC by building an image with given dependencies and running the master DAG
```

```suggestion
test-rc-deps: ## Tests providers RC by building an image with given dependencies and running the master DAG
```

WDYT about a shorter name like `test-rc-deps`